<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDialog</class>
 <widget class="QDialog" name="CustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <widget class="QFrame" name="content_frame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>390</width>
      <height>290</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="drag_win_widget" name="title" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>35</height>
      </rect>
     </property>
     <widget class="QLabel" name="icon_close">
      <property name="geometry">
       <rect>
        <x>363</x>
        <y>10</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="icon_min">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>小ai签票</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_logo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dialog_content" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>390</width>
       <height>210</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="dialog_footer" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>245</y>
       <width>390</width>
       <height>45</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drag_win_widget</class>
   <extends>QWidget</extends>
   <header location="global">components/drag_win_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
